toolkits/global/packages/global-ad/README.md

```diff
@@ -9,11 +9,12 @@ Minimum height set for leaderboard and mpu slots to prevent reflow of layout.
 
 ## Branding
 
-The `global-ad` component currently uses the `DEFAULT` brand only.
+To include `global-ad` in your application, you need to choose **ONE** brand from those available. The `DEFAULT` brand is included in all other brands, and any settings that are not configured will fall back to default.
```

Comment:
Do we need this information in the readme here? 
It seems applicable to how every global package works and perhaps already exists elsewhere?

Reply:
Not sure if this is supposed to be here or not. I've only seen this instruction on the package itself (ie. global-banner).